### DISCUSSION TITLE
Partial automatic wakeup feature to only light up display but don't accept input

### DISCUSSION THREAD

**Discussion by sstark** [InfiniTimeOrg/InfiniTime#1578] [Ideas]
I would wish my watch would just show the time when it wakes up from shake/wrist movement, but not start reacting to touch input. It often happens that it wakes up when I don't want it, which would be fine, if not at the same time it would happen that some menu is opened or some setting changed as well. Preferred would be that I can tell inifinitime to only light up the display on shake induced wakeup. If then I want to also use the touch input I would need to press the button explicitly.
So the possible states would be:

  - Sleep: no touch input, no display
    - Automatic wakeup: display on, but no touch input (if feature enabled) -> press button to enable touch input
    - Automatic wakeup timeout (same duration as normal timeout) -> back to sleep

  or

  - Sleep: no touch input, no display
    - Press button: display and input enabled

I wonder if that would be possible to implement and which areas of the firmware would need to be changed. If somebody could give me some pointer I might give it a try.

**Comment by FintasticMan** [comment]
You could do something similar to #961, just do it on every wake and disable on a single press of the button, rather than what I've done where it's all manual.
